lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts

```diff
@@ -105,12 +106,18 @@ export class AttachFileCloudWidgetComponent extends UploadCloudWidgetComponent i
         return (!this.hasFile || this.multipleOption) && !this.field.readOnly;
     }
 
-    onRemoveAttachFile(file: File | RelatedContentRepresentation | Node) {
+    onRemoveFile(file: File | RelatedContentRepresentation | Node) {
         this.removeFile(file);
         if (file['id'] === this.selectedNode?.id) {
             this.selectedNode = null;
-            this.contentModelFormFileHandler();
         }
+        if(file['id'] === this.lastSentToViewers?.id) {
```

Comment:
what happens if both are not defined? `if (undefined === undefied)` and call the handler?

Reply:
I don't think that at this point `file['id']` can be undefined, but I'll add a check above.